tests/test_doctor.sh

```
@@ -49,12 +49,45 @@ test_doctor_detects_tasks_in_backlog() {
+    output=$("$GITCREW" doctor 2>&1)
+    assert_contains "$output" "uncustomized"
+    assert_contains "$output" "run-tests.sh"
+
+    teardown_sandbox "$sandbox"
+}
+
+# When run-tests.sh no longer has the template TODO, doctor should report configured
+test_doctor_run_tests_configured_when_customized() {
+    local sandbox
+    sandbox=$(setup_sandbox)
+    cd "$sandbox"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)